### ISSUE TITLE
Aggregated by Species: Organize files in folders

### ISSUE THREAD

**Issue by dvenprasad** [AlexsLemonade/refinebio#609]
### Context

This came up when @jaclyn-taroni and I were chatting about illustrations for download files (for docs)

### Problem or idea

Currently, all the files are just dumped in the download folder. The files should be grouped together in folders based on species. 

For example:
![agg-species](https://user-images.githubusercontent.com/15315514/45446641-70981000-b69b-11e8-8d4e-a3e17a4ee3e5.png)

### Solution or next step

Tagging @Miserlou for input on how best move this forward

**Comment by jaclyn-taroni** [comment]
I obtained files from the dev stack this morning. The files are now grouped into folders, but there are some differences from what is in @dvenprasad's illustration here.

**Multispecies aggregated by species example:**
```
multispecies_by_species_example/
├── DANIO_RERIO
│   ├── DANIO_RERIO_metadata.tsv
│   └── DANIO_RERIO.tsv
├── GALLUS_GALLUS
│   ├── GALLUS_GALLUS_metadata.tsv
│   └── GALLUS_GALLUS.tsv
├── LICENSE.TXT
├── metadata.json
├── MUS_MUSCULUS
│   ├── MUS_MUSCULUS_metadata.tsv
│   └── MUS_MUSCULUS.tsv
└── README.md

3 directories, 9 files
``` 
**Single species aggregated by species example:**
```
single_species_by_species_example/
├── DANIO_RERIO
│   ├── DANIO_RERIO_metadata.tsv
│   └── DANIO_RERIO.tsv
├── LICENSE.TXT
├── metadata.json
└── README.md

1 directory, 5 files
```
Specifically:
* There are no species level metadata json files
* The "overall" metadata json file is `metadata.json` rather than `aggregated_metadata.json`

**Comment by dongbohu** [comment]
@jaclyn-taroni and @dvenprasad: I can think of two possible structures of the species-level json file. One is simply an array like this:
```
[ 
  { 1st_sample_metadata }, 
  { 2nd_sample_metadata }, 
  ... 
]
```

The other option is to add some extra fields and make it a dictionary:
```
{
  species_name: "homo sapiens",
  created_at: "...",
  # ... other fields ...

  samples: [
    { 1st_sample_metadata ... }, 
    { 2nd_sample_metadata ... }, 
    ... 
  ]
}
```
Which one do you prefer?

**Comment by dvenprasad** [comment]
I prefer the second one. Even if they misplace their files, there's some context. As for fields, species and created at should be fine for now.

**Comment by dongbohu** [comment]
Thanks @dvenprasad!

**Comment by dongbohu** [comment]
Implemented in #648.
